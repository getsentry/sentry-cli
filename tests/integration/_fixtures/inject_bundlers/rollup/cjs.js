"use strict";var n;n=function(n){throw new Error(n)},function(n,o){!function(n,o){n(o)}(n,o)}(n,"boop");
//# sourceMappingURL=cjs.js.map
