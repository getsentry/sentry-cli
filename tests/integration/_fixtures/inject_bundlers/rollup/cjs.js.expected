"use strict";
!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="0b6cf2c0-b31e-5bb4-9e50-a0c0fd4837db")}catch(e){}}();
var n;n=function(n){throw new Error(n)},function(n,o){!function(n,o){n(o)}(n,o)}(n,"boop");
//# sourceMappingURL=cjs.js.map

//# debugId=0b6cf2c0-b31e-5bb4-9e50-a0c0fd4837db
